index.js

```diff
@@ -131,7 +133,67 @@ alexa.response = function(session) {
     this.sessionObject.clear(key);
     return this;
   };
-
+  this.audioPlayerPlay = function (url, token, playBehavior, offsetInMilliseconds, expectedPreviousToken) {
```

Comment:
`expectedPreviousToken` is required only in case of `playBehavior` is ENQUEUE. Otherwise we don't need it. 
Other params are required in all responses: [look at the table there](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/custom-audioplayer-interface-reference#play-directive)

We can try to assign default values to the arguments and make as you wrote, but I don't sure it'd be convenient.